Source/WebDriver/WebSocketServer.h

```diff
@@ -91,10 +91,10 @@ class WebSocketMessageHandler {
         // Optional connection, as the message might be generated without a connection object available (e.g. inside a method handler).
         // In this case, it gets associated to the connection when sending the message back to the client.
         Connection connection;
-        const char* data { nullptr };
-        size_t dataLength { 0 };
+        const CString payload;
```

Comment:
This is not const, no?

Reply:
As we never modify the string after the message construction (only accessing the c string when sending back the reply), I went with const for it. Is there any reason to make it non-const?